### PR TITLE
Absolute paths for certs

### PR DESCRIPTION
**Pull request by albertrdixon**
Really simple change here to allow for absolute paths to cert files in your kubeconfig.

### CODE REVIEW

**Comment by simon3z** [comment]
@albertrdixon I think that there is something more official than `start_with?('/')` to check if a path is absolute. For example: [this](https://ruby-doc.org/stdlib-1.9.3/libdoc/pathname/rdoc/Pathname.html#method-i-absolute-3F). Also, I have the feeling there was even something simpler to use (can you check?).

**Comment by albertrdixon** [comment]
Fair enough. I was looking for something simple without needed a new object, but I suppose `Pathname` is the way to go.

**Comment by simon3z** [comment]
@moolitayer @enoodle can you review?

**Comment by moolitayer** [comment]
@albertrdixon thanks for the patch. 
Can you please provide a code example of what this is fixing.

**Comment by albertrdixon** [comment]
Sure,

let's say you put all your certs in `/etc/ssl/certs/` then in your kubeconfig you might have `certificate-authority: /etc/ssl/certs/ca.pem`. This patch will fix kubeclient incorrectly trying to find `ca.pem` relative to the kubeconfig.

**Comment by simon3z** [comment]
LGTM :+1: can anyone else review/ack as well? (cc @abonas @moolitayer @enoodle )

**Comment by simon3z** [comment]
@albertrdixon I don't recall, do we have tests for this part? Should we add a test for your new code as well?

File: lib/kubeclient/config.rb [hunk 1]
```diff
@@ -63,6 +64,7 @@ def context(context_name = nil)
     private
 
     def ext_file_path(path)
+      return path if Pathname(path).absolute?
```
Comment (enoodle) [inline lib/kubeclient/config.rb]: Question: shouldn't this be wrapped with `File` as well? (`return File(path) if Pathname(path).absolute?`

Comment (moolitayer) [inline lib/kubeclient/config.rb]: `File.join` below returns a string and that's what we need here as well.

**Comment by moolitayer** [comment]
@albertrdixon thanks. Please squash your two commit;
When someone looks at the log in the future they should see this as one change which this is.

Once you do that I'm good with this change

**Comment by albertrdixon** [comment]
@simon3z no explicit tests for absolute paths, and I'm not sure of an elegant way to add that in. My thinking is that it is sufficient to ensure this does not affect current functionality (relative paths) and - since this is now using `Pathname#absolute?` - assume the `Pathname` will Do The Right Thing.

**Pull request merged**